### ISSUE TITLE
[QUESTION]: Set lam when using Dirichlet loss function

### ISSUE THREAD

**Issue by muammar** [chemprop/chemprop#297]
**What are you trying to do?**
I am trying to predict evidential uncertainty for classification using the Dirichlet loss function. When you try this, you are calling this function: 

https://github.com/chemprop/chemprop/blob/2ef8e3d95194338b4e1f94bbd94ceebaa272b6e5/chemprop/train/loss_functions.py#L245-L262

One of the arguments for that loss is the annealing coefficient `lam` set to 0 by default. That means the loss is not using Kullback-Leibler divergence. What would be the command line flag setting this coefficient to be different from 0?

Thanks.

**Comment by cjmcgill** [comment]
Hi. The argument you want is `--evidential_regularization`. This applies the regularization term for dirichlet and evidential loss functions. 

Originally the dirichlet loss was going to be called evidential_classification as it is considered an evidential loss in the Coley group paper on evidential loss. And then the naming for this parameter made more sense. But we changed the name to dirichlet to distinguish it in the uncertainty functions discussions. Could use a note in the documentation.

**Comment by muammar** [comment]
Thanks, @cjmcgill, that makes sense. I did not read the source code thoroughly. I will proceed to close the report.
